source/core/ut_suite_manager.pks

```diff
@@ -3,14 +3,6 @@ create or replace package ut_suite_manager authid definer is
   function config_package(a_owner_name varchar2, a_object_name varchar2) return ut_test_suite;
 
   procedure config_schema(a_owner_name varchar2);
-
-  procedure run_schema_suites(a_owner_name varchar2, a_reporter in out nocopy ut_reporter, a_force_parse_again boolean default false);
-
-  procedure run_schema_suites_static(a_owner_name varchar2, a_reporter in ut_reporter, a_force_parse_again boolean default false);
-
-  procedure run_cur_schema_suites(a_reporter in out nocopy ut_reporter, a_force_parse_again boolean default false);
-
-  procedure run_cur_schema_suites_static(a_reporter in ut_reporter, a_force_parse_again boolean default false);
   
   procedure run(a_path in varchar2, a_reporter in ut_reporter);
   -- to be implemented
```

Comment:
I would enable this run procedure anyway (even if it only runs for one element of array at the moment).
This way we have a complete API and can start implementing on top of it.
Internal implementation can change for `run( list )` later. But if we're OK with the entry point signature, it should be enabled I think.